src/store-tree/index.ls

```diff
@@ -21,10 +21,7 @@ class StoreTree extends StoreNode
 
 
   $get-error: ->
-    for own key of @_children
-      err = @[key].$get-error!
-      return err if err?
-    null
+    @_children |> values |> find (.$get-error!) |> -> it?.$get-error! or null
```

Comment:
That returns the child whereas I want the actual error. You are correct that the `values` is unnecessary though. The `or null` is needed in order to specifically return null instead of undefined.